### PR TITLE
Anushka/registration

### PR DESCRIPTION
**Pull request by AnushkaKhare-Eng**
# Overview

creating the registration page

# Changes

created the registration page

# Questions & Notes

Are there any open questions? Anything else important we should know?
the page is yet to be formatted properly
# Issue tracking

List any relevant work that still needs to be done. Open and link to issues for them if applicable.
we will open a new PR to fix the aesthetics
Does this close any tasks?
yes Issue 97
# Testing & Demo

image

### CODE REVIEW

File: www/src/routes/register/Registration.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { FC, useState } from 'react';
+
+import StyledSelect from '../../components/Styledselect';
+const Registration: FC = () => {
+    const [year, setYear] = useState('');
+    const [speciality, setSpeciality] = useState('');
+    return (
+        <form>
+            <StyledSelect title='Year' value={year} onChange={setYear} choices={['1', '2', '3', '4', '5', '5+']} />
+            <StyledSelect title='Speciality' value={speciality} onChange={setSpeciality} choices={['Computer Co-op', 'Software Co-op', 'Nano Co-op', 'Computer Trad', 'Nano Trad']} />
```
Comment (cyrusdiego) [inline www/src/routes/register/Registration.tsx:10]: Can you put an "Other" choice

File: www/src/routes/register/Registration.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, { FC, useState } from 'react';
+
+import StyledSelect from '../../components/Styledselect';
+const Registration: FC = () => {
```
Comment (cyrusdiego) [inline www/src/routes/register/Registration.tsx:4]: nit: add newline between line 4 and 3

File: www/src/routes/student/Register.tsx [hunk 3]
```diff
@@ -1,7 +1,27 @@
-import React, { FC } from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import React, { FC, useState } from 'react';
 
-const Register: FC = () => {
-    return <div>Registration page</div>;
-};
+import StyledSelect from '../../components/Styledselect';
+const Registration: FC = () => {
```
Comment (cyrusdiego) [inline www/src/routes/student/Register.tsx:5]: nit: newline between line 4 and 5

File: www/src/routes/register/Registration.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React, { FC, useState } from 'react';
```
Comment (cyrusdiego) [inline www/src/routes/register/Registration.tsx:1]: What's the difference between this file and /students/Registration.tsx in terms of usage?

**Review by cyrusdiego — CHANGES_REQUESTED** [review]
just left a few comments, good work on this :)

File: www/src/routes/register/Registration.tsx (hunk 4 above)
Comment (armiantos) [reply www/src/routes/register/Registration.tsx:1]: This was used during development before @AnushkaKhare-Eng merged from master. This file should be deleted before merging

**Comment by AnushkaKhare-Eng** [comment]
Guys I broke something oops

**Review by cyrusdiego — APPROVED** [review]

**Pull request merged**